web-registry/src/components/molecules/MetadataJSONField.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Field, useFormikContext } from 'formik';
+
+import { Body } from 'web-components/lib/components/typography';
+import InputLabel from 'web-components/lib/components/inputs/InputLabel';
+import ControlledTextField from 'web-components/lib/components/inputs/ControlledTextField';
+
+import { BatchBasicsFormValues } from '../organisms/BatchBasicsForm';
+
+interface FieldProps {
+  name?: string;
+  required?: boolean;
```

Comment:
seems like `required` is not actually being used, ditto in `ProjectSelect`
why not extend formik FieldProps instead?